metricbeat/module/docker/glide.yaml

```diff
@@ -2,3 +2,6 @@ package: github.com/elastic/beats/metricbeat/module/docker
 import:
 - package: github.com/fsouza/go-dockerclient
   version: 23c589186c2a92a8742da55f19aec4997dae5cbb
+import:
```

Comment:
docker client is already mentioned above. You must update the version and overwrite all files from the client with the new commit id to make sure the full client is updated.